src/01-2-basics.js

```diff
@@ -11,6 +11,22 @@
 function generateBandName(clothingColor, lastFoodEaten) {
   // Your code here
   // Initialize bandName to an empty string
+  let bandName = "";
+
   // Function to capitalize the first letter of each word
+  function capitalizeFirstLetter(word) {
+    if (typeof word === "number") {
+       return word.toString();
+     }
+    if (typeof word !== "string" || word === "null") {
+       return "";
+     }
+     return word.charAt(0).toUpperCase() + word.slice(1).toLocaleLowerCase();
```

Comment:
this return statement should be outside the second if statement - then I think you will pass all the tests 💯